src/main/java/homework1/DuplicateInArray.java

```diff
@@ -0,0 +1,21 @@
+package homework1;
+
+import java.util.*;
+
+public class DuplicateInArray {
+
+    public static Set<Integer> findDuplicate(int[] array) {
+
+        HashSet<Integer> duplicates = new HashSet<>();
```

Comment:
Please do the same changes as you did for method's return type - I will explain during Q&A session why it is needed: `Set<Integer> duplicates = new HashSet<>();`

You can also read this article - https://medium.com/javarevisited/oop-good-practices-coding-to-the-interface-baea84fd60d3